### ISSUE TITLE
Kudos

### ISSUE THREAD

**Issue by GugliermoElx** [mbeacom/gitcoin-erc721#211]
i would like to ask for a little advice, ive been getting some Kudos but now is being so difficult to send the to visualize on a wallet, itried metamask, also myether, niftywallet.. and i dont find a way to see the, i have also check the xdai network, and now the kudos are showed on my gitcoin profile but dont see them on wallets. Please some help. @owocki some help please.

here is the info of the last one

0x1da5a74ab37d80b8d2a1e099dfefe2c1e4766d714da4f5859c1bb08b1f5026fe

**Comment by owocki** [comment]
you have to add the contract link to metamask for the network you are on .  checkout https://gitcoin.co/blog/the-future-of-gitcoin-kudos/ which has instructions

**Comment by GugliermoElx** [comment]
yes, i added both networks to metamask as ive been reading that ETH mainet changed to xdai, so i did, but im kind of lost because when i see the kudo in mygitcoin profile says that has been sended to the contract adress, not to the wallet adress.

**Comment by GugliermoElx** [comment]
yes, i added both networks to metamask as ive been reading that ETH mainet changed to xdai, so i did, but im kind of lost because when i see the kudo in mygitcoin profile says that has been sended to the contract adress, not to the wallet adress.

**Comment by GugliermoElx** [comment]
besides metamask im also using niftywallet that is supposed to do it directly but i cant see it either.
